modules/ROOT/pages/oauth2-policies-new.adoc

```diff
@@ -3,25 +3,27 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-To secure an API in Anypoint Platform using OAuth 2.0, you may use the following policies and the matching OAuth 2.0 provider:
+To secure an API in Anypoint Platform using OAuth 2.0, you can use the following policies and the matching OAuth 2.0 provider:
 
-* One of the following policies available in Federated organizations configured with specific OAuth 2.0 providers:
+* Any one of the following policies available in federated organizations, configured with specific OAuth 2.0 providers:
```

Comment:
Meaning of the statement changed. It is no "Any one" but "One", they are exclusive policies. Depending on the organization config, only *one* of the following policies will be available.

Reply:
Kim please verify.

Reply:
They mean virtually the same thing.

Were I to consider my own interpretation of nuance, I would say that:

"any one" implies that you can use one of any of the following
"one" could imply that you must use one particular one of the following, depending on some other circumstance.

Reply:
Note that the sentence is missing its verb.

Reply:
Done! Thanks!